llvm/test/tools/llvm-gsymutil/ARM_AArch64/macho-merged-funcs-dwarf.yaml

```diff
@@ -0,0 +1,740 @@
+# RUN: yaml2obj %s -o %t.dSYM
+# RUN: llvm-gsymutil --convert %t.dSYM --out-file=%t.gSYM
+# RUN: llvm-gsymutil --verify --verbose %t.gSYM | FileCheck --check-prefix=CHECK-GSYM %s
```

Comment:
Does this example rely on the new attribute where each function knows its line table?

Reply:
For this specific test, the [DW_AT_LLVM_stmt_sequence](https://discourse.llvm.org/t/rfc-new-dwarf-attribute-for-symbolication-of-merged-functions/79434) is not needed. 
The script that generates the dSYM yaml is at the bottom (and [in this gist](https://gist.github.com/alx32/b085c7bd886786abeeabd3a42ee22a90)).

Since all the overlapping functions in the dSYM are located in different files, their line table will be separate so we don't need that attribute to differentiate the line entries.